features/waarden.feature

```diff
@@ -0,0 +1,194 @@
+      },
+      {
+        "vastgesteldeWaarde": 176000,
+        "indicatieLopendGeding": true,
+        "waardepeildatum": "2018-01-01"
+      },
+      {
+        "vastgesteldeWaarde": 152000,
+        "waardepeildatum": "2017-01-01"
+      },
+      {
+        "vastgesteldeWaarde": 164000,
+        "waardepeildatum": "2016-01-01"
+      }
+    ]
+    """
+
+  Scenario: WOZ-object heeft nieuwe eigenaar en vorige eigenaar heeft verlaging van de WOZ-waarde na beroep
+    Gegeven WOZ-object met objectnummer "050300024029" heeft met ingang van 01-05-2019 een nieuwe eigenaar R. Krol
+    En de vorige eigenaar W. Jansen heeft beroep ingediend over de WOZ-waarde op 01-01-2018
+    En dat beroep is afgehandeld met de status "uitspraak beroep, vastgestelde waarde veranderd"
+    En de vorige eigenaar W. Jansen heeft hoger beroep aangetekend over de WOZ-waarde op 01-01-2017
+    En dat hoger beroep heeft de status "hoger beroep aangetekend"
+    En de belanghebbende eigenaar heeft bezwaar ingediend over de WOZ-waarde op 01-01-2019
+    En dat bezwaar heeft de status "bezwaar ingediend"
+    En WOZ-object met objectnummer "050300024029" heeft de volgende beschikkingen:
+      | Peildatum  | WOZ-waarde   | Belanghebbende | Ingangsdatum | Datum beschikking |
+      | 01-01-2018 | 190.000 euro | R. Krol        | 01-01-2019   | 27-08-2020        |
+      | 01-01-2019 | 266.000 euro | W. Jansen      | 01-01-2020   | 13-01-2020        |
+      | 01-01-2018 | 223.000 euro | W. Jansen      | 01-05-2019   | 03-06-2019        |
+      | 01-01-2018 | 223.000 euro | R. Krol        | 01-01-2019   | 14-01-2019        |
+      | 01-01-2017 | 189.000 euro | R. Krol        | 01-01-2018   | 15-01-2018        |
+    Als het WOZ object wordt opgevraagd met /wozobjecten/050300024029
+    Dan bevat het antwoord:
+    """
+    "waarden": [
+      {
+        "vastgesteldeWaarde": 266000,
+        "indicatieLopendGeding": true,
+        "waardepeildatum": "2019-01-01"
+      },
+      {
+        "vastgesteldeWaarde": 223000,
+        "waardepeildatum": "2018-01-01"
+      },
+      {
+        "vastgesteldeWaarde": 189000,
+        "indicatieLopendGeding": true,
+        "waardepeildatum": "2017-01-01"
+      }
+    ]
+    """
+
+  Scenario: medebelanghebbende vraagt beschikking met waarde die afwijkt van de waarde van de belanghebbende eigenaar
+    Gegeven WOZ-object met objectnummer "020200143285" heeft de belanghebbende eigenaar K. van Es bezwaar ingediend over de WOZ-waarde op 01-01-2019
+    En dat bezwaar is afgehandeld met de status "bezwaar afgehandeld, vastgestelde waarde veranderd"
+    En medebelanghebbende J. Groenen vraagt daarna een beschikking aan met de waarde op 01-01-2019
+    En WOZ-object met objectnummer "020200143285" heeft de volgende beschikkingen:
+      | Peildatum  | WOZ-waarde   | Belanghebbende | Ingangsdatum | Datum beschikking |
+      | 01-01-2019 | 107.000 euro | J. Groenen     | 01-01-2020   | 30-07-2020        |
```

Comment:
van Groenen krijgt dan toch ook een waarde van 98.000 op de beschikking in juli als de uitspraak in mei is geweest? 
Dacht dat Rijk gisteren zijn dat mede belanghebbenden dezelfde waarden krijgen als deze na een uitspraak is veranderd. Zij kunnen alleen (als zij nog geen beschikking hebben gehad) op een later moment een beschikking opvragen en (binnen een nieuwe bzwaartermijn) bezwaar maken.

Reply:
klopt, kan niet voorkomen dat medebelanghebbende over zelfde periode ander bedrag krijgt